src/main/java/org/apache/commons/cli/Option.java

```diff
@@ -577,7 +577,7 @@ public int getId() {
      *
      * @return the 'unique' Option identifier
      */
-    String getKey() {
+    public String getKey() {
```

Comment:
Hello @Claudenw 
If you make an element `public` or `protected`, you need a `@since` Javadoc tag.